code/game/machinery/computer/communications.dm

```diff
@@ -211,6 +211,8 @@
 			if (bank_account.account_balance < shuttle.credit_cost)
 				return
 			SSshuttle.shuttle_purchased = SHUTTLEPURCHASE_PURCHASED
+			for(var/datum/round_event_control/shuttle_insurance/insurance_event in SSevents.control)
```

Comment:
you left in your debug code

Reply:
least thats what i think this is i have no idea

Reply:
oh i think i get it i guess

Reply:
This is so that the event regains its original weight when you buy a shuttle. See the extended description for the commit.